packages/ord-connect/src/hooks/useSignMessage.tsx

```diff
@@ -26,6 +53,7 @@ export function useSignMessage(): {
           wallet,
```

Comment:
handle address checking above this function call

```
  if (walletAddresses.ordinals !== address && walletAddresses.payments !== address) {
    throw new Error("Address supplied is not connected address");
  }
```

Reply:
updated